src/components/SideInformationsContainer/styles.ts

```diff
@@ -0,0 +1,20 @@
+import styled from "styled-components";
+
+interface Minimizer {
+    isSidebarOpen: boolean;
+    theme: {};
+  }
+
+export const SideInformationsContainer = styled.div<Minimizer>`
+    top: 150px;
+    position: absolute;
+    font-family: "Roboto",sans-serif;
+
+    left: ${({ isSidebarOpen }) => (isSidebarOpen ? "360" : "20")}px;
```

Comment:
Use `365` instead `360` to be align with searchbar

| Before | After |
| :- | :- |
| <img width="147" alt="image" src="https://user-images.githubusercontent.com/61520601/168448157-ea89067e-29bb-431a-9f16-8b8650d9db45.png"> | <img width="124" alt="image" src="https://user-images.githubusercontent.com/61520601/168448176-9029eaff-0fce-4fdb-8cee-cc746be07060.png"> |

```suggestion
    left: ${({ isSidebarOpen }) => (isSidebarOpen ? "365" : "20")}px;
```

Reply:
Okay. I must've calculated wrong.